src/cameras/CubeCamera.js

```diff
@@ -10,7 +10,7 @@ function CubeCamera( near, far, renderTarget ) {
 
 	this.type = 'CubeCamera';
 
-	if ( renderTarget.isWebGLCubeRenderTarget !== true ) {
+	if ( renderTarget.isWebGLCubeRenderTarget === false ) {
```

Comment:
Sorry, these changes are wrong. The idea is to test for the existence of `isWebGLCubeRenderTarget` which can only have a value of `true`. `isWebGLCubeRenderTarget === false` is never the case.

Reply:
> Sorry, these changes are wrong. The idea is to test for the existence of `isWebGLCubeRenderTarget` which can only have the value `true`. `isWebGLCubeRenderTarget === false` is never the case.

Thank you your reply.